app/views/result_filters/location/_form.html.erb

```diff
@@ -34,8 +34,8 @@
               <%= form.label :l_1, "By postcode, town or city", class: "govuk-label govuk-radios__label" %>
             </div>
             <div
-              class="govuk-radios__conditional <%= "govuk-radios__conditional--hidden" unless location_error? %>"
-              id="location-conditional">
+              class="govuk-radios__conditional <%= "govuk-radios__conditional--hidden" unless location_error? || params[:l] == "1" %>"
```

Comment:
I was wondering whether we even need `location_error?` we should show the conditional fields regardless of whether there is an error or not. As long as the user has selected the parent radio button in this case "By postcode.." then the conditional fields should be displayed

Reply:
Good point. 

@SamJamCul and I talked about the `By school` radio, which works correctly because it's checking `params[:query]` but I changed it as you suggested for consistency. 

Ready for another look 👀